upgrade/upgrade.go

```diff
@@ -76,28 +76,33 @@ func Upgrade(kubeconfigPath, currentNodeID string) error {
 		return errors.Wrap(err, "unable to create scheme")
 	}
 
-	if err := upgradeLocalNode(); err != nil {
+	if !upgradeutil.IsUpgradePathSupported(namespace, lhClient) {
+		return fmt.Errorf("unable to upgrade since the current upgrade path is not supported")
+	}
+
+	// If the current Longhorn is already the latest version,
+	// the leader election & the whole upgrade path could be skipped.
+	lhVersionBeforeUpgrade, err := upgradeutil.GetCurrentLonghornVersion(namespace, lhClient)
```

Comment:
For the version variable, you can make it simple as well like `lhCurrentVresion` and `lnUpgradeVersion`.

Reply:
Modified names for these variables. Thanks